README.md

```diff
@@ -1,4 +1,4 @@
-[![CI Status](https://github.com/se-edu/addressbook-level3/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
+[![CI Status](https://github.com/AY2223S2-CS2103T-T11-3/tp/workflows/Java%20CI/badge.svg)](https://github.com/se-edu/addressbook-level3/actions)
```

Comment:
Maybe the path of the link at the back should also be changed to AY2223S2-CS2103T-T11-3/tp instead of se-edu/addressbook-level3 ?

Reply:
MB, didn't notice this! Will update

Reply:
Done!